retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc

```diff
@@ -0,0 +1,447 @@
+. Enable the `Open Log` option and click the *Execute* button.
+
+Once the script completes, select the `Debug Only` filter option to show just the results of each create request. Verify that all fields have been successfully created.
+
+=== Create Manually via Salesforce
+
+To create each of these custom fields and enable visibility for them on page layouts:
+
+. Click on the Salesforce Setup icon.
+. Select the `Object Manager` tab from the Setup home page.
+. Find and select the target Salesforce object.
+. Select the `Fields & Relationships` page.
+. Click the *New* button and create the field as specified above.
+. Once the field has been created, click the `Set Field-Level Security` button (or do these steps later - see below).
+. Enable for desired profile, or tick the checkbox next to `Visible` to enable visibility for the desired profile(s).
+
+Repeat these steps for each custom field in the above table.
+
+[NOTE]
+1. Instead of adjusting permissions for each field as you go, if you are only making them available to one or two profiles, then it may be more efficient to go to `Users > Profiles > {profile} > Object Settings` for each object and adjust the permissions for multiple fields at once.
+1. All `Global_*_Id__c` fields should be visible in layouts (as described above) but made *read-only* to avoid issues with data synchronization.
+
+== Fields to Be Updated
+
+The picklist fields mentioned in the table needs to be updated as per the instructions provided below.
+
+[%header%autowidth.spread]
+|===
+| Salesforce Object Label | Field Label | Field Name | Data Type | To be updated values
+| Order | Order Type | Type | Picklist | ADD_ON, CANCELLATION, INITIAL, JOURNAL, RENEWAL, RETURN, SUBSCRIPTION, UPGRADE
+| Order | Status | Status | Picklist | CANCELLED, CREATED, CONFIRMED, DELIVERED, IN_CART, IN_TRANSIT, INVOICED, LOST, PARTIALLY_SHIPPED, PICKUP_AVAILABLE, PROCESSING, REJECTED, RETURNED
+|===
+
+To update picklist values for a field in the specified object:
+
+. Click on the Salesforce Setup icon.
+. Select the `Object Manager` tab from the Setup home page.
+. Find and select the target Salesforce object.
+. Select the `Fields & Relationships` page.
+. Search for the target field with field label and select it.
+. Under the header `Picklist Values`, select `New`.
+. Add the respective `To be updated values` from the table with each picklist value on its own line.
+. Click `Save`.
+
+Repeat these steps for each field in the above table.
+
+If the fields under `Custom Fields` header were created manually, update the picklist values for the fields in the table as per the instructions above.
+
+[%header%autowidth.spread]
+|===
+| Salesforce Object Label | Field Label | Field Name | Data Type | To be updated values
+| Account | Customer Status | Type | Picklist | Prospect, Onboarding, Active, Inactive, Closed, Deceased, Delinquent, Dormant
```

Comment:
Should the items in the Values column be all caps?

Reply:
It is not in upper-case